backend/app/utils/jsonl_formatter.py

```
@@ -0,0 +1,354 @@
+    'StreamingFormatter',
+    'FormatterConfig',
+    'FieldMapping',
+    'OutputFormat',
+    'create_compact_formatter',
+    'create_pretty_formatter',
+    'create_debug_formatter',
+    'create_type_based_formatter',
+    'format_blocks_compact',
+    'format_blocks_pretty',
+    'format_parse_result'
+] 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)